Do not use -Wno-format-contains-nul. clang doesn't understand it and prints
a warning:

warning: unknown warning option '-Wno-format-contains-nul' [-Wunknown-warning-option]

while gcc 4.2 prints an error:

cc1: error: unrecognized command line option "-Wno-format-contains-nul"
--- configure.orig	2018-07-16 17:29:47.000000000 -0500
+++ configure	2018-07-24 16:04:50.000000000 -0500
@@ -18368,1 +18368,1 @@
-  CFLAGS="$CFLAGS -Wno-format-contains-nul -fno-strict-aliasing"
+  CFLAGS="$CFLAGS -fno-strict-aliasing"
